man/systemd.mount.xml

```diff
@@ -318,6 +318,44 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>x-systemd.makefs</option></term>
+
+        <listitem><para>The file system or swap structure will be intialized
+        on the device. If the device is not "empty", i.e. it contains any signature,
+        the operation will be skipped. It is hence expected that this option
+        remains set even after the device has been initalized.</para>
+
+        <para>Note that this option can only be used in
+        <filename>/etc/fstab</filename>, and will be ignored when part of the
+        <varname>Options=</varname> setting in a unit file.</para>
```

Comment:
maybe explain how to do it with unit files...

Reply:
The new systemd-growfs@.service man page does. It does not explain all the details, but I think we don't need to do that. First the details might change yet, and second, we don't do that for many other helper units like that, for example our fsck@ units.

Reply:
ok, that was more of a case of mentionning the man page so people don't think it's impossible to do without fstab, just done differently

I guess advanced users can just look at what is generated for fstab entries anyway...

Reply:
There's a plan to add a framework for allowing partitions to be created and resized, and it'll probably also support this part. So I'd rather not document all the details right now, since they're likely to change.